scipy/optimize/_differentialevolution.py

```diff
@@ -132,6 +134,29 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
         ``np.std(pop) <= atol + tol * np.abs(np.mean(population_energies))``,
         where and `atol` and `tol` are the absolute and relative tolerance
         respectively.
+    updating : str, optional
```

Comment:
`updating : {'immediate', 'deferred'}, optional`. and then remove the 4 lines below